business/checkers/k8shttproute_checker.go

```diff
@@ -11,11 +12,12 @@ import (
 const K8sHTTPRouteCheckerType = "k8shttproute"
 
 type K8sHTTPRouteChecker struct {
-	K8sHTTPRoutes    []*k8s_networking_v1.HTTPRoute
-	K8sGateways      []*k8s_networking_v1.Gateway
-	Namespaces       models.Namespaces
-	RegistryServices []*kubernetes.RegistryService
-	Cluster          string
+	K8sHTTPRoutes      []*k8s_networking_v1.HTTPRoute
```

Comment:
Just a minor thing, it would be possible to alphabetize?

Reply:
ah right